tests/slr_jest/blockEditor/hook-callbacks.spec.js

```diff
@@ -29,26 +31,26 @@ describe('hook-callbacks', () => {
 		jest.resetAllMocks();
 	});
 
-	describe('setSeatTypeForTicket', () => {
-		it('should call dispatch with appropriate arguments', () => {
-			const clientId = 'client-id-1';
+	describe("setSeatTypeForTicket", () => {
+		it("should call dispatch with appropriate arguments", () => {
+			const clientId = "client-id-1";
 			dispatch.mockReturnValue({
 				setTicketSeatTypeByPostId: jest.fn(),
 			});
 
 			setSeatTypeForTicket(clientId);
 			expect(dispatch).toHaveBeenCalledWith(storeName);
 			expect(dispatch().setTicketSeatTypeByPostId).toHaveBeenCalledWith(
-				clientId
+				clientId,
 			);
 		});
 	});
 
-	describe('filterSetBodyDetails', () => {
-		it('should modify body', () => {
-			const seatType = 'seat-type-uuid-1';
-			const layoutId = 'layout-uuid-1';
-			const eventCapacity = '40';
+	describe("filterSetBodyDetails", () => {
```

Comment:
is these changes your IDE's changes ? Mine doesn't catch those with the linter.

Reply:
If you're talking about style and formatting changes, I'm using prettier and jslint.